platforms/Windows/shared/swift.en-us.wxl

```diff
@@ -3,6 +3,7 @@
 <WixLocalization xmlns="http://wixtoolset.org/schemas/v4/wxl" Culture="en-US">
   <String Id="DowngradeError" Value="A newer version of [ProductName] is already installed." />
   <String Id="Bld_ProductName" Value="Swift Build Tools" />
```

Comment:
note for reviewers. `Bld_ProductName` is used for feature selection in the options page; where as  `BldAsserts_ProductName` is the product name in the msi. the user would not see the `BldAsserts_ProductName`, but adding it for future proofing when we have other variants.

Reply:
Should we consider putting that in as a comment?

Reply:
I can do that once we have a second variant. it feels a moot point now